client/gcs_service.go

```diff
@@ -77,13 +78,13 @@ func (s Service) GetOrgPublicKeys(ctx context.Context, env, bucketName, path str
 	if pubKey != "" {
 		return []string{pubKey}, nil
 	}
-	resp, err := s.store.ListOjbect(ctx, bucketName, path)
```

Comment:
If it's meant for abstraction/local, keep it private. 
Also i don't see any logic handled under `GetObjList` except just returning error and repeated here as well, why do we need this method?

Reply:
The GetObjList is added beause i dont wnat client to directly call store.ListObject